<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QMainWindow" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VHF Voice Communication System</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":ico/phoneBlack">
    <normaloff>Resource/phoneBlack.png</normaloff>Resource/phoneBlack.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView::item::selected{
	background-color:blue;
	color:white;
}

QTableView::item{
	height:50px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="groupTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>自动播放</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>483</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="Volume" name="volume" native="true">
                <property name="minimumSize">
                 <size>
                  <width>350</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="qTableWidget">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="im_broad_btn">
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>即时播报</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">broad_grp</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="plan_broad_btn">
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>计划播报</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">broad_grp</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="plan_widget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_b_time">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string> 播报开始时间</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateTimeEdit" name="task_start_time">
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="displayFormat">
                 <string>yyyy/MM/dd HH:mm</string>
                </property>
                <property name="calendarPopup">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_times">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string> 次数</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="times">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_gap">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>间隔时间</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="time_interval">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>单位：分钟</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="title">
              <string>播报岸站</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <item>
               <widget class="QTreeView" name="treeView"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>150</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>300</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="title">
              <string>操作</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="DeleteBtn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>33</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background:qlineargradient(spread:pad, x1:0.79, y1:0.983, x2:1, y2:0.0109091, stop:0 rgba(209, 199, 178, 255), stop:1 rgba(255, 255, 255, 255))</string>
                </property>
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ChangeBtn">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>33</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background:qlineargradient(spread:pad, x1:0.79, y1:0.983, x2:1, y2:0.0109091, stop:0 rgba(209, 199, 178, 255), stop:1 rgba(255, 255, 255, 255))</string>
                </property>
                <property name="text">
                 <string>修改</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="play">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>33</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background:qlineargradient(spread:pad, x1:0.79, y1:0.983, x2:1, y2:0.0109091, stop:0 rgba(209, 199, 178, 255), stop:1 rgba(255, 255, 255, 255))</string>
                </property>
                <property name="text">
                 <string>试听</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="SendBtn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>33</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background:qlineargradient(spread:pad, x1:0.79, y1:0.983, x2:1, y2:0.0109091, stop:0 rgba(209, 199, 178, 255), stop:1 rgba(255, 255, 255, 255))</string>
                </property>
                <property name="text">
                 <string>生成播报任务</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>待播放列表</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="task_del">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1380</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Volume</class>
   <extends>QWidget</extends>
   <header location="global">volume.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="broad_grp"/>
 </buttongroups>
</ui>
